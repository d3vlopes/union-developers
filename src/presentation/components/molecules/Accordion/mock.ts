import { AccordionProps } from '.'

export const accordionMock: AccordionProps = {
  title: 'O que é o Union?',
  content: `
    <p>O Union é um ambiente prático e imersivo próximo do encontrado nas empresas, desafiando seus membros a resolver problemas reais, com cronogramas e prazos de entrega.</p> 
    <p>No Union, você desenvolve as skills mais procuradas pelo mercado, trabalha com ferramentas e metodologias utilizada pelas empresas, além de ganhar experiência trabalhando com outras pessoas de varias partes do Brasil.</p> 
    <p>Você vai vivenciar na prática como é o dia a dia de um desenvolver em uma empresa e ficar mais preparado para conquistar sua vaga.</p>
  `,
}
