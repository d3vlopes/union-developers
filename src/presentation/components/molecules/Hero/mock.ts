import { HeroProps } from './types'

type MockType = 'default' | 'long-title' | 'long-description'

const defaultProps: HeroProps = {
  title:
    '<h1>Quer se tornar um desenvolvedor <strong>diferenciado?</strong></h1>',
  description:
    'Aqui você vivencia na prática como é o dia a dia de um desenvolvedor em uma empresa e se prepara para enfrentar os desafios do mercado de forma prática e imersiva!',
  buttonText: 'Quero participar',
  buttonTarget: '#join',
  backgroundMobile:
    'https://media.graphassets.com/j1rOxE97RsCbeyFrMU4z',
  backgroundDesktop:
    'https://media.graphassets.com/66rHD5NBR0K9lcIF9t62',
  logoUrl: 'https://media.graphassets.com/zRjyxnD5RyGdG8xVpll8',
  logoAlt: 'Description logo',
}

export const heroMockFactory: Record<MockType, HeroProps> = {
  default: {
    ...defaultProps,
  },
  'long-title': {
    ...defaultProps,
    title:
      '<h1>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Placeat, alias consequatur. Totam vero omnis praesentium, repellat dicta veniam illum, unde a sed doloribus quis non dolores autem ab nulla neque!</h1>',
  },
  'long-description': {
    ...defaultProps,
    description: `
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Placeat, alias consequatur. Totam vero omnis praesentium, repellat dicta veniam illum, unde a sed doloribus quis non dolores autem ab nulla neque!
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Placeat, alias consequatur. Totam vero omnis praesentium, repellat dicta veniam illum, unde a sed doloribus quis non dolores autem ab nulla neque!
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Placeat, alias consequatur. Totam vero omnis praesentium, repellat dicta veniam illum, unde a sed doloribus quis non dolores autem ab nulla neque!
    `,
  },
}
