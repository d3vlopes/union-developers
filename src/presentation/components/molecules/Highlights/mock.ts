import { HighlightModel } from '@/domain/models'

export const highlightsMock: HighlightModel[] = [
  {
    id: '1',
    icon: {
      url: 'https://media.graphassets.com/qssZQEUSEaAQoM6oznCw',
      alt: 'rocket',
    },
    title: 'Crescimento profissional',
    description:
      'Eleve o seu conhecimento, aperfeiçoe suas habilidades e alavanque o seu potencial com uma experiência intensa que te ajudará a ser um profissional ainda melhor.',
  },
  {
    id: '2',
    icon: {
      url: 'https://media.graphassets.com/BN50QqvcR67hdjNyuqkA',
      alt: 'pencil and scale',
    },
    title: 'Aumente sua caixa de ferramentas',
    description:
      'Descubra novas metodologias e encare desafios que agregam em seu conhecimento profissional, essenciais para avançar na sua trajetória.',
  },
  {
    id: '3',
    icon: {
      url: 'https://media.graphassets.com/WR1yF2qGRmm8jWLCyw0D',
      alt: 'like',
    },
    title: 'Problemas Reais',
    description:
      'Trabalhe em projetos reais com equipes reais para sair da sua zona de conforto e acelerar seu aprendizado.',
  },
  {
    id: '4',
    icon: {
      url: 'https://media.graphassets.com/2S5Pdk5oShOJUlIz8p7X',
      alt: 'planet',
    },
    title: 'Ambiente Imersivo',
    description:
      'Espaço completo que te prepara, conecta com o mercado e permite absorver conteúdos fundamentais para seguir a sua jornada profissional.',
  },
  {
    id: '5',
    icon: {
      url: 'https://media.graphassets.com/KHCFnl40RyKQqOx6Bjuz',
      alt: 'bulb on',
    },
    title: 'Experiência',
    description:
      'Com os processos fundamentais e domínio das ferramentas te dá a experiência necessária para tomar as melhores decisões no mundo real.',
  },
  {
    id: '6',
    icon: {
      url: 'https://media.graphassets.com/XH5lk8IJQ1Cz1M2AE1Lu',
      alt: 'game controller',
    },
    title: 'Vivência na Prática',
    description:
      'Vivencie o dia a dia do mercado com um time de diferentes mentalidades para solucionar problemas em diversas áreas.',
  },
  {
    id: '7',
    icon: {
      url: 'https://media.graphassets.com/eBMuKBvTLulsEbZySGwt',
      alt: 'star',
    },
    title: 'Desenvolvimento de Skills',
    description:
      'Aprenda a comunicar suas ideias em um ambiente profissional e melhorar suas habilidades de gerenciamento de tempo para se destacar em sua carreira.',
  },
  {
    id: '8',
    icon: {
      url: 'https://media.graphassets.com/sanArFzbQUipTnjRVl78',
      alt: 'person',
    },
    title: 'Mentoria',
    description:
      'No Union você irá ter do lado um profissional experiente que vai te guiar e mostrar os atalhos para que você consiga evoluir mais rápido em menos tempo.',
  },
]
