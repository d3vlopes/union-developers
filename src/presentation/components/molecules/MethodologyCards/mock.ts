import { MethodologyModel } from '@/domain/models'

export const methodologyCardsMock: MethodologyModel[] = [
  {
    id: '1',
    title: 'Ativa',
    description:
      'É uma metodologia de aprendizado que coloca o aluno como protagonista, onde ele é o responsável por ir atrás dos conhecimentos necessários, ficando o professor como um mentor, que vai mostrar o caminho e indicar os melhores conteúdos, que vai guiar o estudo e os métodos, mas também deixar espaço para o aluno explorar.',
  },
  {
    id: '2',
    title: 'Andragogia',
    description:
      'É uma metodologia de ensino para adultos, que tem mais liberdade e colaboração, permitindo que o aluno decida quando, como e o que estudar, tornando o processo de aprendizagem mais sólido e eficaz. É uma metodologia de ensino mais prática, que estimula o aluno a resolver problemas reais, que estão relacionado ao seu interesse.',
  },
  {
    id: '3',
    title: 'Ágeis',
    description:
      'A metodologia ágil é uma metodologia que busca entregar valor a todo momento, estimula uma maior colaboração entre o time, permitindo que tenha uma visão mais transparente, tendo liberdade para fazer escolhas e tomar decisões, também estimula a melhora continua entre os membros, processos e da qualidade dos entregáveis.',
  },
]
