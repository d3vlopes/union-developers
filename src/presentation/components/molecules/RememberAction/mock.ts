import { RememberActionProps } from '.'

export const rememberActionMock: RememberActionProps = {
  title:
    'Vivencia na prática, ambiente imersivo, desafios e muito mais!',
  subtitle: 'Inscreva-se já! Vagas limitadas',
  buttonText: 'Inscrever-se agora',
  buttonTarget: '#subscribe',
  logoUrl: 'https://media.graphassets.com/WayysMB8RQyrgcahBMQs',
  logoAlt: 'Logo description',
}
