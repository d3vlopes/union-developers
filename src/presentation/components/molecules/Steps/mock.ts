import { StepModel } from '@/domain/models'

export const stepsMock: StepModel[] = [
  {
    id: '1',
    title: 'Pré-inscrição',
    description:
      'Inicie a sua inscrição preenchendo o nosso formulário',
    number: 1,
  },
  {
    id: '4',
    title: 'Entrevista',
    description:
      'Nessa fase iremos bater um papo para que possamos conhecer melhor você',
    number: 4,
  },
  {
    id: '3',
    title: 'Desafio técnico',
    description: 'Após a análise você receberá um desafio prático',
    number: 3,
  },
  {
    id: '2',
    title: 'Análise do formulário',
    description: 'O nosso time irá analisar suas informações',
    number: 2,
  },
]
