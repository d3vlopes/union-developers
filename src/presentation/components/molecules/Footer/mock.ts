import { generateId } from '@/utils'

import { logoMock } from '@/presentation/components/atoms/Logo/mock'

import { FooterProps } from '.'

export const footerMock: FooterProps = {
  logo: {
    imgUrl: logoMock.imgUrl,
    alt: logoMock.alt,
  },
  socialLinks: [
    {
      id: generateId(),
      icon: {
        imgUrl: 'https://media.graphassets.com/4AjPcOSSuK4kIKUv8Mdg',
        alt: 'Um ícone de envelope representando o email do Union',
      },
      label: 'Email',
      target: 'contato.uniongroup@gmail.com',
    },
    {
      id: generateId(),
      icon: {
        imgUrl: 'https://media.graphassets.com/y4KMQdygSH6uZx46pxjU',
        alt: 'Ícone com as letras in representando a logo do Linkedin',
      },
      label: 'Linkedin',
      target: 'www.linkedin.com/in/company/uniondevelopers',
    },
    {
      id: generateId(),
      icon: {
        imgUrl: 'https://media.graphassets.com/4lDgbi8FQpq8QYSiranx',
        alt: 'Um gatinho representando o logo do Github',
      },
      label: 'Github',
      target: 'www.github.com/union-group',
    },
  ],
  companyName: 'Union Developers',
  companySlogan:
    'O Union proporciona experiências para desenvolvedores em inicio de carreira, desafiando seus membros a resolver problemas reais.',
  navigationMenuLinks: [
    {
      id: generateId(),
      title: 'Saiba mais',
      links: [
        {
          label: 'Sobre',
          target: '#about',
        },
        {
          label: 'Depoimentos',
          target: '#testimonials',
        },
      ],
    },
    {
      id: generateId(),
      title: 'Dúvidas',
      links: [
        {
          label: 'FAQ',
          target: '#questions',
        },
        {
          label: 'Como funciona',
          target: '#steps',
        },
      ],
    },
  ],
}
