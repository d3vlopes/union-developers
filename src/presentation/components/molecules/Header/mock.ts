import { logoMock } from '@/presentation/components/atoms/Logo/mock'

import { HeaderProps } from './types'

export const headerMock: HeaderProps = {
  logoUrl: logoMock.imgUrl,
  logoAlt: logoMock.alt,
  navigationLinks: [
    {
      label: 'Sobre',
      target: '#about',
    },
    {
      label: 'Benefícios',
      target: '#highlights',
    },
    {
      label: 'Etapas',
      target: '#steps',
    },
    {
      label: 'Dúvidas',
      target: '#questions',
    },
  ],
  buttonText: 'Junte-se a nós',
  buttonTarget: '#join',
}
