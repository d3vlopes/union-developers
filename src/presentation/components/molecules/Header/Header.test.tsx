import { fireEvent, render, screen } from '@/__tests__/helpers'

import { headerMock } from './mock'

import { Header } from '.'

const {
  getByLabelText,
  getByText,
  getAllByRole,
  getByRole,
  getByTestId,
} = screen

describe('<Header />', () => {
  it('should render logo', () => {
    render(<Header {...headerMock} />)

    const logo = getByRole('img')

    expect(logo).toBeInTheDocument()
  })

  it('should render navigation links', () => {
    render(<Header {...headerMock} />)

    const { navigationLinks } = headerMock

    navigationLinks.forEach(async (item) => {
      const navigationAnchor = getByLabelText(item.label)
      const navigationItem = getByText(item.label)

      expect(navigationAnchor).toHaveAttribute(
        'href',
        `#${item.target}`,
      )
      expect(navigationItem).toBeInTheDocument()
    })
  })

  it('should render button', () => {
    render(<Header {...headerMock} />)

    const button = getByText(headerMock.buttonText)

    expect(button).toBeInTheDocument()
  })

  it('should render anchor under button with href for button target value', () => {
    render(<Header {...headerMock} />)

    const anchor = getByTestId('button-anchor')

    expect(anchor).toHaveAttribute('href', headerMock.buttonTarget)
  })

  it('should handle the open/close mobile menu ', () => {
    render(<Header {...headerMock} />)

    const [hamburgerButton] = getAllByRole('button')
    const navigation = getByLabelText('Navegação principal')

    expect(hamburgerButton).toHaveAttribute(
      'aria-label',
      'Abrir menu',
    )
    expect(navigation.getAttribute('aria-hidden')).toBe('true')
    expect(navigation).toHaveStyle({
      opacity: 0,
      visibility: 'hidden',
    })

    fireEvent.click(hamburgerButton)

    expect(hamburgerButton).toHaveAttribute(
      'aria-label',
      'Fechar menu',
    )
    expect(navigation.getAttribute('aria-hidden')).toBe('false')
    expect(navigation).toHaveStyle({
      opacity: 1,
      visibility: 'visible',
    })
  })
})
