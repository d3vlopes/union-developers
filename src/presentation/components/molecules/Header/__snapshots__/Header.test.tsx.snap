// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Header /> > should match snapshot 1`] = `
.c8 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  border-radius: 5px;
  width: 100%;
  background: none;
  -webkit-transition: filter 0.3s ease-in-out;
  transition: filter 0.3s ease-in-out;
  box-shadow: 0px 4px 4px 0px rgba(0,0,0,0.25);
  overflow: hidden;
  height: auto;
}

.c8 span {
  color: #FFFFFF;
  font-size: 2.2rem;
  font-family: Poppins,apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif;
  -webkit-letter-spacing: 1.5px;
  -moz-letter-spacing: 1.5px;
  -ms-letter-spacing: 1.5px;
  letter-spacing: 1.5px;
  font-weight: 700;
  z-index: 2;
}

.c8:disabled {
  opacity: 0.7;
  cursor: not-allowed;
}

.c8::before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  border-radius: 5px;
  background: linear-gradient(225deg,#0048FE 0%,#851D86 100%);
  -webkit-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
  z-index: 1;
}

.c1 {
  max-width: 125.6rem;
  width: 100%;
  height: auto;
  margin: 0 auto;
  padding: 1.6rem 1.2rem;
}

.c0 {
  position: fixed;
  top: 0;
  left: 0;
  background: #121119;
  -webkit-transition: all 0.35s ease-in-out;
  transition: all 0.35s ease-in-out;
  width: 100%;
  height: auto;
  z-index: 50;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  height: auto;
  margin: 0 auto;
}

.c5 {
  width: 100%;
  height: 0;
  padding: 0;
  opacity: 0;
  visibility: hidden;
  overflow: hidden;
  -webkit-transition: all 0.35s ease;
  transition: all 0.35s ease;
}

.c5 .active::after {
  content: '';
  background: linear-gradient(225deg,#0048FE 0%,#851D86 100%);
  height: 1.5px;
  border-radius: 10px;
  width: 100%;
  position: absolute;
  bottom: -5.5px;
  left: 0;
}

.c5 ul {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  gap: 2rem;
  padding: 4rem 0 2rem 2rem;
}

.c5 ul a {
  font-family: Poppins,apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif;
  font-size: 1.8rem;
  font-weight: 600;
  line-height: 2.2rem;
  color: #E2E2E2;
  text-rendering: optimizeLegibility;
  -webkit-transition: all 0.35s ease-in-out;
  transition: all 0.35s ease-in-out;
  position: relative;
}

.c5 ul a:hover,
.c5 ul a .active {
  background: linear-gradient(225deg,#0048FE 0%,#851D86 100%);
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
}

.c5 ul a li {
  padding: 0;
  margin: 0;
  border: none;
}

.c4 {
  position: relative;
  width: 2.5rem;
  height: 2.5rem;
  border: none;
  outline: none;
  background: none;
  -webkit-transform: rotate(0deg);
  -ms-transform: rotate(0deg);
  transform: rotate(0deg);
  -webkit-transition: 0.35s ease-in-out;
  transition: 0.35s ease-in-out;
  display: block;
  opacity: 1;
  visibility: visible;
}

.c4 span {
  display: block;
  position: absolute;
  width: 100%;
  height: 2px;
  left: 0;
  border: none;
  outline: none;
  opacity: 1;
  border-radius: 4px;
  background: #E2E2E2;
  -webkit-transform: rotate(0deg);
  -ms-transform: rotate(0deg);
  transform: rotate(0deg);
  -webkit-transition: 0.25s ease-in-out;
  transition: 0.25s ease-in-out;
}

.c4 span:nth-child(1) {
  top: 0;
  left: 0;
  width: inherit;
}

.c4 span:nth-child(2) {
  top: 0.8rem;
}

.c4 span:nth-child(3) {
  top: 0.8rem;
}

.c4 span:nth-child(4) {
  top: 1.6rem;
  width: inherit;
  left: 0;
}

.c6 button {
  display: none;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  height: 5.9rem;
  padding: 1rem 0.5rem;
  margin-left: 2rem;
  margin-bottom: 3rem;
  max-width: 18rem;
}

.c6 button span {
  font-size: 1.8rem;
  min-width: 16.7rem;
  min-height: 2.7rem;
}

@media (min-width:1024px) {
  .c8:hover:not(.c7:disabled) {
    border-right: 2px solid #0048FE;
    border-left: 2px solid #851D86;
    background-image: linear-gradient(to right,#851D86,#0048FE), linear-gradient(to right,#851D86,#0048FE);
    background-size: 100% 2px;
    background-position: 0 100%,0 0;
    background-repeat: no-repeat;
    background-clip: padding-box;
  }

  .c8:hover:not(.c7:disabled):hover {
    -webkit-filter: brightness(0.9);
    filter: brightness(0.9);
  }

  .c8:hover:not(.c7:disabled)::before {
    left: 100%;
  }
}

@media (max-width:1100px) {
  .c3 img {
    width: 22rem;
  }
}

@media (min-width:1024px) {
  .c1 {
    padding: 2.2rem 1.6rem;
  }
}

@media (min-width:1400px) {
  .c1 {
    padding: 2.2rem 0;
  }
}

@media (min-width:1024px) {
  .c0 {
    height: 11.4rem;
  }
}

@media (min-width:1024px) {
  .c2 {
    height: 6.9rem;
  }
}

@media (min-width:1024px) {
  .c5 {
    width: initial;
    opacity: initial;
    visibility: initial;
    overflow: initial;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }

  .c5 ul {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    gap: 1.6rem;
    padding: 0;
  }

  .c5 .active::after {
    content: '';
    background: linear-gradient(225deg,#0048FE 0%,#851D86 100%);
    height: 1.5px;
    width: 100%;
    position: absolute;
    bottom: -46px;
    left: 0;
  }
}

@media (min-width:1400px) {
  .c5 ul a {
    font-size: 2rem;
  }
}

@media (min-width:1024px) {
  .c4 {
    display: none;
    visibility: hidden;
    opacity: 0;
  }
}

@media (min-width:1024px) {
  .c6 button {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    margin-bottom: initial;
    min-width: 18.6rem;
    margin-left: 0;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (min-width:1400px) {
  .c6 button {
    min-width: 29.9rem;
  }
}

<div>
  <header
    class="c0"
  >
    <div
      class="c1"
    >
      <nav
        class="c2"
      >
        <div
          class="c3"
        >
          <a
            href="#"
          >
            <img
              alt="Um octógono com três pontos, representando a união dos desenvolvedores"
              src="https://res.cloudinary.com/dbnq26wqe/image/upload/v1688490290/union/logo_desktop_uppabl.svg"
            />
          </a>
        </div>
        <button
          aria-expanded="false"
          aria-label="Abrir menu"
          class="c4"
          type="button"
        >
          <span />
          <span />
          <span />
          <span />
        </button>
        <div
          aria-hidden="true"
          aria-label="Navegação principal"
          class="c5"
          role="navigation"
        >
          <ul>
            <a
              aria-label="Sobre"
              href="#about"
            >
              <li
                class=""
              >
                Sobre
              </li>
            </a>
            <a
              aria-label="Pilares"
              href="#pillars"
            >
              <li
                class=""
              >
                Pilares
              </li>
            </a>
            <a
              aria-label="Benefícios"
              href="#highlights"
            >
              <li
                class=""
              >
                Benefícios
              </li>
            </a>
            <a
              aria-label="Etapas"
              href="#steps"
            >
              <li
                class=""
              >
                Etapas
              </li>
            </a>
            <a
              aria-label="Depoimentos"
              href="#testimonials"
            >
              <li
                class=""
              >
                Depoimentos
              </li>
            </a>
            <a
              aria-label="Dúvidas"
              href="#questions"
            >
              <li
                class=""
              >
                Dúvidas
              </li>
            </a>
          </ul>
        </div>
        <div
          class="c6"
        >
          <a
            data-testid="button-anchor"
            href="#join"
          >
            <button
              class="c7 c8"
            >
              <span>
                Junte-se a nós
              </span>
            </button>
          </a>
        </div>
      </nav>
    </div>
  </header>
</div>
`;
